src/js/components/explorer/detail/visualization/toolbar/BreakdownDropdown.jsx

```diff
@@ -34,12 +34,18 @@ export default class BreakdownDropdown extends React.Component {
 
         this.toggleMenu = this.toggleMenu.bind(this);
         this.pickItem = this.pickItem.bind(this);
+        this.setWrapperRef = this.setWrapperRef.bind(this);
+        this.handleClickOutside = this.handleClickOutside.bind(this);
     }
 
     componentWillMount() {
         this.prepareOptions(this.props);
     }
 
+    componentDidMount() {
+        document.addEventListener('mousedown', this.handleClickOutside);
```

Comment:
Instead of always listening for this event (which could add to our overhead), what if we only added the listener when the dropdown is expanded (we have access to the `expanded` state here) and then removed when the dropdown collapses again and/or the component unmounts?